go/vt/mysqlctl/backup.go

```diff
@@ -90,7 +91,7 @@ var (
 // - uses the BackupStorage service to store a new backup
 // - shuts down Mysqld during the backup
 // - remember if we were replicating, restore the exact same state
-func Backup(ctx context.Context, cnf *Mycnf, mysqld MysqlDaemon, logger logutil.Logger, dir, name string, backupConcurrency int, hookExtraEnv map[string]string) error {
+func Backup(ctx context.Context, cnf *Mycnf, mysqld MysqlDaemon, logger logutil.Logger, dir, name string, backupConcurrency int, hookExtraEnv map[string]string, backupTime time.Time) error {
```

Comment:
@enisoc what do you think of pushing the time of the backup into this function instead of passing it in? We have too many places now where we have to do `time.Now().UTC()`.

Reply:
I'd prefer to still be able to pass it in, if that's not too cumbersome. In vtbackup, I "fake" the backup time to a time in the past to make it more accurate:

https://sourcegraph.com/github.com/vitessio/vitess@3fea94e2c9e4e2fca2078b712632bee67ea0a5f3/-/blob/go/cmd/vtbackup/vtbackup.go#L281-284

Reply:
:+1: